storybook/storybook-docs/src/introduction.md

```diff
@@ -2,81 +2,103 @@
 
 # Amsterdam Design System
 
-## Wat is het
+## What is it
```

Comment:
I think we can remove this, it's conventional that the first paragraph after the h1 explain what you're looking at